src/webview/git-import/gitImportLoader.ts

```diff
@@ -69,7 +78,24 @@ export class Command {
             });
             vscode.window.showInformationMessage('Selected Component added to the workspace.');
         }
+        if (!isWorkspaceFolder(workspacePath)) {
+            vscode.workspace.updateWorkspaceFolders(vscode.workspace.workspaceFolders ? vscode.workspace.workspaceFolders.length : 0, null, { uri: appendedUri });
+        }
+    }
+}
+
+
+function isWorkspaceFolder(uri: vscode.Uri): boolean {
```

Comment:
You should be using [workspace](https://code.visualstudio.com/api/references/vscode-api#workspace).getWorkspaceFolder() instead of writing your own function.